### PR TITLE
CNV-35029: Release note NEW

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.15

Issue: [CNV-35029](https://issues.redhat.com/browse/CNV-35029)

Link to docs preview: Note in the [Storage section](https://71911--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-storage) of the 4.15 Release Notes 

QE review: Complete

### CODE REVIEW

**Comment by aspauldi** [comment]
Hi @aglitke,  I don't know that I'll be able to complete the doc story that corresponds to this RN in time for the 4.15 release. Catherine asked me to write the RN just in case.

Can you take a look and see what you think? If I can get the related doc story done, I'll add an xref from the RN, but, if not, the RN will be all that is doc'd for this feature in 4.15.

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -101,6 +101,8 @@ This release adds new features and enhancements related to the following compone
 === Storage
 
 //CNV-35029: PVC annotation to explain host-assisted cloning
+* By default, the Customization Data Importer (CDI) chooses the most efficient strategy for cloning a data volume: CSI volume cloning or smart cloning. Both cloning strategies have certain prerequisites for use. If the prerequisites are not met, host-assisted cloning, a less efficient cloning method, is available.
```
[the remark this answers is not in the document]
Comment (aspauldi) [reply virt/release_notes/virt-4-15-release-notes.adoc:104]: @aglitke, thanks for the correction on CDI and the release note text!

**Review by jpeimer — APPROVED** [review]

**Review by skopacz1 — APPROVED** [review]
LGTM!

**Pull request merged**